test/specs/homework2.js

```diff
@@ -0,0 +1,113 @@
+    });
+/**
+ * Verify destination and check-in/ and check-out dates are as user selected
+ * 
+ * 1. Launch hotels.com
+ * 2. Type "man" in destination
+ * 3. Select "Manila" from auto-suggestion
+ * 4. Select Aug 20 to Sep-5 as check-in and check-out respectively
+ * 5. Click Search button
+ * 6. Verify destination has Manila
+ * 7. Verify check-in date in Aug-20
+ * 8. Verify check-out date in Sep-5
+ * 
+ * 
+ */
+
+    it.only('Verify destination and check-in/ and check-out dates are as user selected', async () => {
+        
+        // Launch hotels.com
+        await browser.url('https://www.hotels.com/');
+        await browser.pause(4000);
+        
+        // Type "man" in destination
+        await $("//button[@aria-label='Going to']").click();
+        await browser.pause(4000);
+        
+        await $('#destination_form_field').setValue('Man');
+        await browser.pause(4000);
+        
+        // Select "Manila"
+        const selectCity = await $$("//div[@class='truncate']//strong");
+        
+        for (const cityText of selectCity) {
+            const citySuggestion = await cityText.getText();
+            if (citySuggestion.toLowerCase().localeCompare('Manila'.toLowerCase()) === 0) {
+                await cityText.click();
+                break;
+            }
+        }
+        await browser.pause(4000);
+
+        await $('#date_form_field-btn').click();
+        await browser.pause(4000);
+
+        await $('//button[@data-stid="date-picker-paging"]').click();
+        await browser.pause(4000);
+        
+        // Select Aug 20 to Sep-5 as check-in and check-out respectively
+        const allAugDateElements = await $$('//h2[text()="August 2022"]/following-sibling::table//button[not(@disabled)]');
+        for (const dateElement of allAugDateElements) {
+            const date = await dateElement.getAttribute('data-day');
+            if (date.localeCompare("20") === 0) {
+                await dateElement.click();
+                break;
+            }
+        }
+        await browser.pause(4000);
+        
+        const allSepDateElements = await $$('//h2[text()="September 2022"]/following-sibling::table//button[not(@disabled)]');
+        for (const dateElement of allSepDateElements) {
+            const date = await dateElement.getAttribute('data-day');
+            if (date.localeCompare("5") === 0) {
+                await dateElement.click();
+                break;
+            }
+        }
+        await browser.pause(4000);
+        
+        // Click Search button
+        await $('//button[@data-stid="apply-date-picker"]').click();
+        await browser.pause(4000);
+        
+        await $('//button[@id="submit_button"]').click();
+        await browser.pause(4000);
+
+        // Verify destination has Manila
+        const destination = await $("//div[@data-stid='hotels-destination']//div//button[@type='button']").getText();
+        console.log(`Destination -> ${destination}`);
```

Comment:
Printing in log is not necessary for automation, bcz there no one gonna check log for thousands of test cases rather use expect() always for verification. It gonna tell abt failure.